.github/workflows/cargo-update.yml

```diff
@@ -24,7 +20,7 @@ jobs:
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v4
```

Comment:
It might be time to replace this externally controlled action with something else. Options include, but may not be limited to:
* Fork the `peter-evans/create-pull-request` repo
* Reference the `peter-evans/create-pull-request` repo by SHA instead of `v4` so it can't change out from under the tag
* Use the [`actions/github-script`](https://github.com/actions/github-script) instead
  * It's still an action, but at least it's maintained by GitHub
  * The [octokit documentation for "Pulls -> Create"](https://octokit.github.io/rest.js/v18#pulls-create) is the source of what would go into that action's "asynchronous function call"
* Use the [GitHub Pulls REST API](https://docs.github.com/en/rest/pulls/pulls#create-a-pull-request) directly
  * Examples there use `curl` and the `gh` CLI, both of which are available [on the runners](https://github.com/actions/virtual-environments/blob/ubuntu20/20220605.1/images/linux/Ubuntu2004-Readme.md)

Reply:
Replaced with `actions/github-script`. Thanks for the recommendation.